src/ice.c

```diff
@@ -3997,6 +3997,20 @@ static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data
 	}
 
 	if(pc && pc->do_transport_wide_cc && medium) {
+		/* Simulcast layers that have not received data will not have a recorded peer SSRC. */
+		/* Pick the first layer that has a peer SSRC */
+		guint32 ssrc_peer = 0;
+		for(int i = 0; i < 3; i++) {
+			if(medium->ssrc_peer[i] != 0) {
+				ssrc_peer = medium->ssrc_peer[i];
+				break;
+			}
+		}
+		if (ssrc_peer == 0) {
+			JANUS_LOG(LOG_WARN, "No peer SSRC, cannot send transport-wide CC feedback\n");
```

Comment:
I wouldn't display a warning here: in case it happens, this is going to spam the logs a lot, since TWCC feedback is very frequent (5 to 10 times per second by default). I'd make it a `LOG_HUGE` instead.